pkg/core/runtime/runtime.go

```diff
@@ -212,6 +212,6 @@ func (rc *runtimeContext) MeshValidator() core_managers.MeshValidator {
 	return rc.mv
 }
 
-func (rc *runtimeContext) ShutdownCh() <-chan struct{} {
-	return rc.shutdownCh
+func (rc *runtimeContext) AppContext() context.Context {
```

Comment:
Nit: godoc comment would be nice to distinguish app context from other kinds of context.